install/kubernetes/helm/istio/test-values/values-istio-dns-cert.yaml

```diff
@@ -0,0 +1,7 @@
+global:
```

Comment:
I was actually thinking to move these to `test-values/values-e2e.yaml`, and then delete `chiron/main_test.go`, move your  `dns_cert_test.go` to test/security.

@howardjohn I think it's fine to introduce these values into the shared values file, right?

Reply:
Added a TODO item to investigate whether this test can be moved to test/security. 1.4 deadline is approaching and the test placement can be optimized later.

Reply:
Had a discussion with @JimmyCYJ on merging the tests. Merging multiple tests in the same Istio deployment may interfere with each other and may cause the test results undeterministic. To make the tests run faster, the tests should run in parallel.

Reply:
@diemtvu thoughts?